predicators/spot_utils/spot_utils.py

```diff
@@ -846,47 +854,28 @@ def hand_movement(self,
                 BODY_FRAME_NAME, "hand")
             qw, qx, qy, qz = body_T_hand.rot.w, body_T_hand.rot.x,\
                 body_T_hand.rot.y, body_T_hand.rot.z
-        elif angle_45:
-            # Set downward place rotation as a quaternion.
-            qw = np.cos(np.pi / 8)
-            qx = 0
-            qy = np.sin(np.pi / 8)
-            qz = 0
         else:
             # Set downward place rotation as a quaternion.
-            qw = np.cos(np.pi / 4)
-            qx = 0
-            qy = np.sin(np.pi / 4)
-            qz = 0
+            qw, qx, qy, qz = angle
         flat_body_Q_hand = geometry_pb2.Quaternion(w=qw, x=qx, y=qy, z=qz)
 
-        # Make the arm pose RobotCommand
-        if use_object_location:
-            # Get graph_nav to body frame.
-            state = self.get_localized_state()
-            gn_origin_tform_body = math_helpers.SE3Pose.from_obj(
-                state.localization.seed_tform_body)
-
-            # Apply transform to fiducial pose to get relative body location.
-            assert isinstance(obj, Object)
-            tag_id = obj_name_to_apriltag_id[obj.name]
-            body_tform_fiducial = gn_origin_tform_body.inverse(
-            ).transform_point(apriltag_id_to_obj_poses[tag_id][0],
-                              apriltag_id_to_obj_poses[tag_id][1],
-                              apriltag_id_to_obj_poses[tag_id][2])
-            hand_x, hand_y, hand_z = [
-                body_tform_fiducial[0], body_tform_fiducial[1], self.hand_z
-            ]
+        if not relative_to_default_pose:
+            x = params[0]  # dx hand
+            y = params[1]
+            z = params[2]
         else:
-            hand_x, hand_y, hand_z = [self.hand_x, self.hand_y, self.hand_z]
-        # Build a position to move the arm to (in meters, relative to and
-        # expressed in the gravity aligned body frame).
-        assert params[0] >= -0.5 and params[0] <= 0.5
-        assert params[1] >= -0.5 and params[1] <= 0.5
-        assert params[2] >= -0.25 and params[2] <= 0.25
-        x = hand_x + params[0]  # dx hand
-        y = hand_y + params[1]
-        z = hand_z + params[2]
+            x = self.hand_x + params[0]  # dx hand
+            y = self.hand_y + params[1]
+            z = self.hand_z + params[2]
+
+        x_cliped = np.clip(x, 0.3, 0.9)
+        y_cliped = np.clip(y, -0.5, 0.5)
+        z_cliped = np.clip(z, 0.2, 0.7)
+        self.relative_move((x - x_cliped), (y - y_cliped), 0.0)
```

Comment:
Three things:
1. Add a comment explaining what you're doing here
2. Make the magic number constants you're using be obvious constants at the top of the file
3. (nit) i think 'clipped' is misspelled